cfme/tests/configure/test_default_views_infra.py

```diff
@@ -34,9 +33,8 @@ def setup_a_provider():
     'VMs': Vm,
     'My Services': MyService,
     'Catalog Items': CatalogItem,
-    'VMs & Instances': 'service_vms_instances',
-    'Templates & Images': 'service_templates_images'
-}
+    'VMs & Instances': VmsInstances,
+    'Templates & Images': TemplatesImages}
```

Comment:
:+1:  but consider making this like it was done by psav in the following commit
https://github.com/ManageIQ/integration_tests/pull/4003/files#diff-707235ec81985bc8d13d47291b348740